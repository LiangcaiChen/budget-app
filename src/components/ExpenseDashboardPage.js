import React, {Component} from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpensesListFilters';

class ExpenseDashboardPage extends Component {
    render() {
        return (
            <div>
                <ExpenseListFilters />
                <ExpenseList />
            </div>
        )
    }
}

export default ExpenseDashboardPage;