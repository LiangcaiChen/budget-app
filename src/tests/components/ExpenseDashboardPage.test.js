import React from 'react';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';
import { shallow } from 'enzyme';
import '../setupTests';
import toJson from 'enzyme-to-json';

test('should render ExpenseDashboardPage correnctly', ()=>{
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
});