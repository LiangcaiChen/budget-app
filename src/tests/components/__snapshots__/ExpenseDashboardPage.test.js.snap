// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage correnctly 1`] = `
<div>
  <Connect(ExpensesListFilters) />
  <Connect(ExpenseList) />
</div>
`;
